Assets/Scripts/Collectables/HealthBasedDropper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Actor;
+using UnityEngine;
+
+namespace Collectables
+{
+    /// <summary>
+    /// A dropper that drops drops based on the health of the entity it's bound to.
+    /// </summary>
+    public class HealthBasedDropper : Dropper
+    {
+        /// <summary>
+        /// The entity that this dropper is attached to. Used to keep track of health.
+        /// </summary>
+        public Living entity;
+        
+        /// <summary>
+        /// The health points at which to drop drops.
+        /// </summary>
+        public int[] healthMarkers;
+
+        private IList<int> sortedHealthMarkers;
+
+        private void Start()
+        {
+            // If there are no health markers for which to drop drops, destroy this object.
+            if (healthMarkers.Length <= 0)
+            {
+                Destroy(this);
```

Comment:
Same here. Is this `Destroy` intended?

Reply:
Yes, I want to just destroy the dropper (not the entire enemy) if it's no longer needed.